packages/onboarding/styles/variables.scss

```diff
@@ -0,0 +1,10 @@
+@import './base-styles';
+
+// Reusable style variables
+$onboarding-header-height: $header-height;
+$onboarding-footer-height: $header-height;
```

Comment:
Take note that this was originally hardcoded to `64px`. This is now inherited directly from core WP which was a value of `60px`. This ensures header height consistency from gutenboarding to block editor.